skyportal/handlers/api/instrument.py

```diff
@@ -71,6 +73,21 @@ def post(self):
                 return self.error('`field_region` is required with field_data')
             regions = Regions.parse(field_region, format='ds9')
 
+            if type(field_data) is str:
+                field_data = pd.read_table(StringIO(field_data), sep=",").to_dict(
+                    orient='list'
+                )
+
+            if not all(
+                k in field_data
+                for k in [
+                    'ID',
+                    'RA',
+                    'Dec',
+                ]
+            ):
```

Comment:
Again styling choice but I feel this is more readable (especially after linting).
```suggestion
            if not {'ID', 'RA', 'Dec'}.issubset(field_data):
```